sklearn/naive_bayes.py

```diff
@@ -880,6 +880,7 @@ def __init__(
 
     def __sklearn_tags__(self):
         tags = super().__sklearn_tags__()
+        tags.input_tags.sparse = True
```

Comment:
It seems that this is shared with `_BaseDiscreteNB`. So we can define it once only in the base class since the validation of `fit` is located there.

Reply:
done in 705c414